src/integrations/watchers/option-wpseo-watcher.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Yoast\WP\SEO\Integrations\Watchers;
+
+use Yoast\WP\SEO\Integrations\Integration_Interface;
+use Yoast\WP\SEO\Conditionals\No_Conditionals;
+
+/**
+ * Watcher for the wpseo option.
+ *
+ * Represents the option wpseo watcher.
+ */
+class Option_Wpseo_Watcher implements Integration_Interface {
+
+	use No_Conditionals;
+
+	/**
+	 * Initializes the integration.
+	 *
+	 * This is the place to register hooks and filters.
+	 *
+	 * @return void
+	 */
+	public function register_hooks() {
+		\add_action( 'update_option_wpseo', [ $this, 'check_semrush_option' ], 10, 2 );
+	}
+
+	/**
+	 * Checks if the SEMrush integration is disabled; if so, deletes the tokens.
+	 *
+	 * We delete the tokens it the SEMrush integration is disabled, no matter if
```

Comment:
Should this be "We delete the tokens **if**", instead of "it"?

Reply:
Will fix it.